site/fetchers/blog-posts/index.js

```diff
@@ -55,9 +55,9 @@ const getPostsForTag = params =>
 
 /* eslint-disable camelcase */
 const getPosts = (tags: Array<string>) =>
-  (Promise.all(tags.map(topicId => getPostsForTag({ topic_id: topicId }))).then(flatten): Promise<
-    any,
-  >);
+  (Promise.all(tags.map(topicId => getPostsForTag({ topic_id: topicId }))).then(
+    flatten,
+  ): Promise<any>);
```

Comment:
I wonder about these formatting changes - are we using different prettier configs (I'm sure I've touched these files recently)

Reply:
I think you may be right. My linter also seems to want me to make everything a `.ts` file.

Reply:
Potentially you need to flow and prettier plugins? They should read the config files in the repo.